google/cloud/bigtable/table_admin_test.cc

```diff
@@ -614,7 +576,7 @@ TEST_F(TableAdminTest, DropAllRows) {
   EXPECT_CALL(*client_, DropRowRange(_, _, _)).WillOnce(Invoke(mock));
 
   // After all the setup, make the actual call we want to test.
-  tested.DropAllRows("the-table");
+  EXPECT_TRUE(tested.DropAllRows("the-table").ok());
```

Comment:
If this assertion fails, there won't be anything very useful in the logs since the `Status` object is not logged. In other cases I think we've been doing something like:

```c++
auto x = foo.DoFoo();
EXPECT_TRUE(x.ok()) << x;
```

Yes, that's a bit of a PITA to do in all cases. I filed https://github.com/googleapis/google-cloud-cpp/issues/1980 to create an ASSERT_OK() that will do the Right Thing for StatusOr and Status objects, but it doesn't exist yet.

Reply:
Done with `ASSERT_STATUS_OK`.